code/obj/item/rcd.dm

```diff
@@ -219,9 +219,6 @@ Broken RCD + Effects
 			if (RCD_MODE_LIGHTTUBES)
 				boutput(user, "Changed mode to 'Light Tube Fixture'")
 
-		// Gonna change this so it doesn't shit sparks when mode switched
-		// Just that it does it only after actually doing something
-		//src.shitSparks()
 		src.UpdateIcon()
 		return
```

Comment:
```suggestion
```
god why are there so many of these everywhere

Reply:
very important, must be certain the proc returns, defensive programming